test/support/config_factory.ex

```diff
@@ -47,6 +45,15 @@ defmodule ExAwsConfigurator.Factory.Config do
 
         %{name => topic_config}
       end
+
+      defp build_attributes(%{fifo_queue: true}) do
+        [
+          content_based_deduplication: true,
+          fifo_queue: true
+        ]
+      end
+
+      defp build_attributes(_attrs), do: []
```

Comment:
```suggestion
      defp build_fifo_queue_attributes(%{fifo_queue: true}) do
```

ficou estranho build_attributes retornando [], mudar o nome pra build_fifo_queue_attributes fica mais claro ;)